packages/voter-dapp/src/ActiveRequests.js

```diff
@@ -408,7 +420,8 @@ function ActiveRequests({ votingAccount, votingGateway, snapshotContract }) {
       const newCommitKey = toVotingAccountAndPriceRequestKey(
         votingAccount,
         pendingRequests[index].identifier,
-        pendingRequests[index].time
+        pendingRequests[index].time,
+        pendingRequests[index].ancillaryData
```

Comment:
Wait, why do you use `pendingRequests[index].ancillaryData` here but hardcode `0x` above?

Reply:
Yea I think i made some mistakes here, good catch